pkg/router/template/router_test.go

```diff
@@ -13,6 +13,7 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/apimachinery/pkg/util/sets"
 
+	"github.com/google/go-cmp/cmp"
```

Comment:
nit - I wish this had openshift/api before any k8s.io imports.

Reply:
I am not sure what exactly you want here

Reply:
can please specify in detail ?

Reply:
I wish L16+17 were above L12.  Not a big deal.

Reply:
ok done